javascript/exercise5/js/select_box_doc.js

```diff
@@ -0,0 +1,32 @@
+function SelectBoxDoc(list1, list2, addButton, deleteButton) {
+  this.leftList = list1;
```

Comment:
list1 ??
list2 ??

These names are not correct
You can right
function SelectBoxDoc(leftList, rightList, addButton, deleteButton) {
this.rightList = rightList;
this.leftList = leftList;
}